File: LP_book/Cap6_2DMatriz/matriz2D_e.cpp
#include <stdio.h>

int main(int argc, char const *argv[])
{
    int matrizA[10] = {2,2,2,2,2,2,2,2,2,2}; 
    int matrizC[10][3];
    int x,y,i,j,fat = 1;

    for (y = 0; y < 3; y++)
    {
        for ( x = 0; x < 10; x++)
        {
            if (y == 0)
            {
                matrizC[x][y] = 5 + matrizA[x];
            }
            else if (y == 1)
            {
                for (i = matrizA[x]; i > 1; i--)
                {
                    fat *= i;
                }
                matrizC[x][y] = fat;
                fat = 1;
            }
            else if (y == 2)
            {
                matrizC[x][y] = matrizA[x] * matrizA[x];
            }
            
            
            
        }
        
    }

    for ( y = 0; y < 3; y++)
    {
        for ( x = 0; x < 10; x++)
        {
            printf("%d\n", matrizC[x][y]);
        }
        
    }
    
    
    return 0;
}

File: LP_book/Cap6_2DMatriz/matriz2D_e.py
from makeMatriz import matrizBiOrdinaria,matrizUniOrdinaria,printMatrizBi

matrizA = matrizUniOrdinaria(12, 10)
matrizB = matrizUniOrdinaria(12, 20)
matrizC = matrizBiOrdinaria(2, 12, 5)


for y in range(2):
    for x in range(12):
        if y == 0:
            matrizC[x][y] = 2 * matrizA[x]
        else:
            matrizC[x][y] = matrizB[x] - 5

printMatrizBi(matrizC, 5)
